Raindrops/StandardRaindrop.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DigitalRain.Raindrops
+{
+    class StandardRaindrop : IRaindrop
+    {
+        // IRaindrop
+        public char[] SymbolPool { get; private set; }
+        public char Symbol { 
+            get { return _symbol; }
+            set {
+                _symbol = value; 
+                _isDrawingNewSymbol = true;
+            } 
+        }
+        public int Lifespan { get { return 255; } }
+        public double LifeRemaining { get; private set; }
+        public static Color DefaultColor { get { return Color.GreenYellow; } }
+
+        private char _symbol;
+        private readonly double _decayRate;
+        private readonly Vector2 _position;
+        private Color _symbolColor;
+        private Vector2 _symbolCenter;
+        private bool _isDrawingNewSymbol;
+
+        public StandardRaindrop(double decayRate, Vector2 initialPosition, Color symbolColor)
+        {
+            SymbolPool = SymbolPools.EnglishAlphanumericUppercase();
+            Symbol = GetSymbolFromPool();
+            LifeRemaining = (double)Lifespan;
```

Comment:
Lifespan shouldn't be a byte.  It should be a "wall clock" time.  Then we can do timePassed/lifespan to get the elapsed time as a percentage of the lifespan, multiply that by 255, and finally subtract from 255 to get the alpha.  We'll also have to clamp to make sure we stay between 0 and 255.